File: app/src/main/java/closer/vlllage/com/closer/api/models/GroupInviteResult.java
package closer.vlllage.com.closer.api.models;

import closer.vlllage.com.closer.store.models.GroupInvite;

public class GroupInviteResult extends ModelResult {
    public String name;
    public String group;

    public static GroupInvite from(GroupInviteResult result) {
        GroupInvite groupInvite = new GroupInvite();
        groupInvite.setId(result.id);
        groupInvite.setGroup(result.group);
        groupInvite.setName(result.name);
        groupInvite.setUpdated(result.updated);
        return groupInvite;
    }
}

File: app/src/main/java/closer/vlllage/com/closer/api/models/GroupInviteResult.kt
package closer.vlllage.com.closer.api.models

import closer.vlllage.com.closer.store.models.GroupInvite

class GroupInviteResult : ModelResult() {
    var name: String? = null
    var group: String? = null

    companion object {

        fun from(result: GroupInviteResult): GroupInvite {
            val groupInvite = GroupInvite()
            groupInvite.id = result.id
            groupInvite.group = result.group
            groupInvite.name = result.name
            groupInvite.updated = result.updated
            return groupInvite
        }
    }
}
